pkgs/build-support/rust/defaultCrateOverrides.nix

```diff
@@ -0,0 +1,6 @@
+{ pkgconfig, sqlite, openssl, ... }:
+
+{
+  libsqlite3-sys = attrs: { buildInputs = [ pkgconfig sqlite ] ++ (attrs.buildInputs or []); };
+  openssl-sys = attrs: { buildInputs = [ pkgconfig openssl ] ++ (attrs.buildInputs or []); };
```

Comment:
Whatever format we start with in this file will be likely followed blindly until it gets way out of hand. Can you reformat these to be multiple lines? Example:

```
  libsqlite3-sys = attrs: {
    buildInputs = [ pkgconfig sqlite ]
      ++ (attrs.buildInputs or []);
  };
```

Reply:
The single line looks more amenable to reading imho, easy to copy a line, move it up and down in alphabetical order using an editor. 2cw

Reply:
I will not become this big. Must package won't need `overriding`. 
Our `defaultGemConfig` file is  only 342 lines long.

Reply:
well okay then, if other language level pkg managers use a similar indentation convention then best to stick with it, but at least do this after the merge?